proxy/logging/LogField.cc

```diff
@@ -132,6 +168,13 @@ LogField::LogField(const char *name, const char *symbol, Type type, MarshalFunc
 
   m_time_field = (strcmp(m_symbol, "cqts") == 0 || strcmp(m_symbol, "cqth") == 0 || strcmp(m_symbol, "cqtq") == 0 ||
                   strcmp(m_symbol, "cqtn") == 0 || strcmp(m_symbol, "cqtd") == 0 || strcmp(m_symbol, "cqtt") == 0);
+
+  if (m_milestone_map.empty()) {
```

Comment:
This should really be in a local function or method, since it's used multiple times.